doc/man3/SSL_SESSION_get_time.pod

```diff
@@ -66,7 +66,9 @@ of the session.
 SSL_SESSION_get_time_ex() and SSL_SESSION_get_timeout() return the currently
 valid values.
 
-SSL_SESSION_set_time_ex() and SSL_SESSION_set_timeout() return 1 on success.
+SSL_SESSION_set_time_ex() returns time on success.
+
+SSL_SESSION_set_timeout() returns 1 on success.
 
 If any of the function is passed the NULL pointer for the session B<s>,
```

Comment:
```suggestion
All functions return 0 if the session B<s> is NULL.
```

Reply:
This is pre-existing. It looks like there's a lot of work to get this whole file sorted, shall we just take the trivial changes here and worry about the rest later?